chain-abci/src/app/query.rs

```diff
@@ -170,26 +170,6 @@ impl<T: EnclaveProxy> ChainNodeApp<T> {
                     "app state not found",
                 );
             }
-            "account" => {
-                let account_address = StakedStateAddress::try_from(_req.data.as_slice());
-                if let (Some(state), Ok(address)) = (&self.last_state, account_address) {
-                    let (account, _proof) =
-                        get_with_proof(&self.storage, state.staking_version, &address);
-                    match account {
-                        Some(a) => {
-                            resp.value = a.encode();
-                            // TODO: inclusion proof
-                        }
-                        None => {
-                            resp.log += "account lookup failed: account not exists";
-                            resp.code = 1;
-                        }
-                    }
-                } else {
-                    resp.log += "account lookup failed (either invalid address or node not correctly restored / initialized)";
-                    resp.code = 3;
-                }
-            }
```

Comment:
is there a need to do a breaking change? the original default behaviour should identical to "staking" path without any parameters? it could possibly be a non-breaking change with `"staking" | "account" => {...` pattern match... it could log some deprecation warning for "account" path that it'd be removed in the future